lib/rmt/cli/products.rb

```diff
@@ -62,7 +63,7 @@ def change_product(target, set_enabled)
 
     if set_enabled
       products.each do |product|
-        extensions = Product.recommended_extensions(product.id).to_a
+        extensions = all_modules ? Product.free_and_recommended_modules(product.id) : Product.recommended_extensions(product.id).to_a
```

Comment:
can the to_a on the false branch be omitted, or should it be added to the true branch for semantic clarity? Both methods return ActiveRecord::Relation.